src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp

```diff
@@ -99,9 +99,12 @@ SubCmdScan::execute(const SubCmdOptions& _options) const
     auto udev = xrt_core::get_userpf_device(device_id);
     auto vbnv = xrt_core::device_query<xrt_core::query::rom_vbnv>(udev);
     auto bdf = xrt_core::device_query<xrt_core::query::pcie_bdf>(udev);
-    std::cout << "[" << device_id << "]: "
-              << xrt_core::query::pcie_bdf::to_string(bdf) << " "
-              << vbnv << "\n";
+    std::cout << "[" << device_id << "]: ";
+    std::string bdf_val = xrt_core::query::pcie_bdf::to_string(bdf);
+    if(bdf_val != std::string("NA"))
```

Comment:
The original code was focused on PCIe device.  It should have instead requested a platform, see if it was a PCIe device, and then extracted this information.  

Let me look into the code code early next week to create a template for you to follow.